.github/workflows/documentation.yaml

```diff
@@ -21,36 +21,31 @@ jobs:
         with:
           ref: main
 
+      - name: Install terraform-docs
+        run: |
+          curl -sSLo ./terraform-docs.tar.gz https://terraform-docs.io/dl/v0.18.0/terraform-docs-v0.18.0-$(uname)-amd64.tar.gz
+          tar -xzf terraform-docs.tar.gz
+          chmod +x terraform-docs
+          mv terraform-docs /usr/local/bin/terraform-docs
+
       - name: Render Terraform docs inside the README.md files of examples
-        id: terraform-docs-examples
-        uses: terraform-docs/gh-actions@v1.2.0
-        with:
-          config-file: templates/examples-terraform-docs.yaml
-          recursive: true
-          recursive-path: examples
-          output-file: README.md
-          args: --sort-by required
-          indention: 2
-          git-push: "false"
+        run: |
+          find examples -type d -mindepth 1 -maxdepth 1 | while read dir; do
+            terraform-docs -c "./templates/examples-terraform-docs.yaml" --output-file "README.md" "$dir"
```

Comment:
@thogarty I couldn't find a corresponding attribute (in GHA) to support the last parameter in terraform docs command( "$dir" ) which defines inside which directory the terraform-docs command would run. Therefore, I decided to use this approach of using running a script.

Reply:
I support the approach. Still leverages terraform-docs just not directly with their GHA job. 

I was wondering why a pull request wasn't created at the the end of the action though? https://github.com/equinix/terraform-equinix-fabric/actions/runs/10322345359/job/28577358771

In the action you ran it should create a PR with the updates that it made during the earlier jobs.